src/components/Sidebar/SideBar.tsx

```diff
@@ -12,13 +13,29 @@ type Props = {
 
 const Background = styled.div`
   background-color: ${Color.Cinder};
+  display: flex;
+  flex-flow: column nowrap;
+`;
+
+const Plus = styled(plus)`
+  fill: ${Color.White};
+  margin-right: 5px;
 `;
 
 const SideBar = (props: Props) => {
   return (
     <Background className={props.className}>
       <div>Top part</div>
-      <div>Bottom part</div>
+      <div
+        css={`
+          margin: auto auto 15px 10px;
+        `}
+      >
+        <Button>
+          <Plus />
+          <span css={{ color: Color.White }}>Add New...</span>
```

Comment:
If you look closely at the button, you can see that the text and the icon are not well aligned:
![Screenshot_2020-05-08 React App](https://user-images.githubusercontent.com/64152453/81444433-6a5f7d00-9177-11ea-8ae2-686d6c89a77f.png)

My recomendation would be to add a small flexbox and use it to center it. You can learn more about that [here](https://css-tricks.com/centering-css-complete-guide/).
```suggestion
          <div css={{ /* make me a flexbox to center my content */ }}>
            <Plus />
            <span css={{ color: Color.White }}>Add New...</span>
          </div>
```